.prettierignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# Lighthouse
+.lighthouseci
```

Comment:
I think this line isn't necessary because `prettier` claims to ignore files in `.gitignore` by default https://prettier.io/docs/en/ignore.html#:~:text=By%20default%20prettier,it%20is%20run.